pkg/cluster/scripts/fixetcd.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# See for more information: https://docs.openshift.com/container-platform/4.10/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html
+
+remove_peer_members() {
+    echo "${PEER_PODS}"
+	for p in ${PEER_PODS}; do
+		echo "Attempting to get ID for pod/${p}"
+		members="$(oc rsh -n openshift-etcd -c etcdctl "pod/${p}" etcdctl member list -w json --hex true)"
+		id="$(jq -r --arg node "$DEGRADED_NODE" '.members[] | select( .name == $node).ID' <<< "$members")"
+		echo "id: ${id:-Not Found}"
+		if [[ -n $id ]]; then
+			echo "rshing into pod/${p} now to remove member id $id"
+			oc rsh \
+				-n openshift-etcd \
+				-c etcdctl \
+				"pod/${p}" etcdctl member remove "$id"
+		else
+			echo "${DEGRADED_NODE} id not found in etcd member list for pod ${p}"
+		fi
+	done
+}
+
+create_sym_links() {
+    jq_lib1="/usr/lib64/libjq.so.1"
+    jq_lib2="/usr/lib64/libonig.so.5"
+
+    if [[ ! -f $jq_lib1 ]]; then
+        ln -s "/host${jq_lib1}" "$jq_lib1"
+    fi
+    if [[ ! -f $jq_lib2 ]]; then
+        ln -s "/host${jq_lib2}" "$jq_lib2"
+    fi
+}
```

Comment:
Can you leave a comment for why this is needed? It's worth noting we are `chroot /host` at this point, on a random worker node.

Reply:
@SudoBrendan I'll add a comment. The reason is that the mounted host binary `jq` won't run without the shared libraries being in the roots path, not `/host/`.

Reply:
Let me know if that comment needs refined